vm/vmimpl/freebsd.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package vmimpl
+
+import (
+	"io"
+	"time"
+)
+
+// DiagnoseFreeBSD sends the debug commands to the given writer which
+// is expected to be connected to a paniced openbsd kernel.  If kernel
```

Comment:
I guess you mean `a paniced FreeBSD kernel`.

Reply:
Also the past tense of panic is panicked :)

Reply:
@zxombie Can you update the comment to address Ed's and my comment?